### ISSUE TITLE
Add link to module require debugger in  require "error" message.

### ISSUE THREAD

**Issue by triptych** [observablehq/feedback#176]
**Is your feature request related to a problem? Please describe.**
You try to require some snippet or code and it's not a module or something.  You just get an error message, but no "help". If I did not know about the  "module require debugger" I would be lost. Please add a link to this debugger when the UI detects that the user is trying  and failing to include some code.
**Describe the solution you'd like**
When the user hits a "require error" of some kind - add something like "for help with requiring other code see https://observablehq.com/@observablehq/module-require-debugger"
**Describe alternatives you've considered**
I just have it bookmarked and use it every day.
**Additional context**
New users will not have any idea how to get something to work on their own, and might leave in frustration. Offering up some contextual help for a very common issue would help newbies learn to use Observable.

**Comment by CobusT** [comment]
Yeah, that is a great idea. We have been thinking about ways to do that!

**Comment by abitrolly** [comment]
An exception handler around https://github.com/d3/d3-require should make it.
